float64 x
float64 y
float64 w
float64 h
float64 conf
